### PR TITLE
netpbm, switch to official super_stable uri

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Tried adding the TEST() case, but that ends up in $PATH errors not finding library.

**Comment by Begasus** [comment]
Not checked on 32bit (gcc2) yet.

File: media-libs/netpbm/netpbm-10.86.42.recipe [hunk 1]
```diff
@@ -231,16 +231,16 @@ COPYRIGHT="2001 411toppm: Steve Allen
...
+CHECKSUM_SHA256="d4d2ca69d82b8d8518aee1d0a2f400664324afa8237859386764a41784d497f6"
 PATCHES="netpbm-$portVersion.patchset"
 
 ARCHITECTURES="all"
 SECONDARY_ARCHITECTURES="x86_gcc2 x86"
 
 PROVIDES="
 	netpbm$secondaryArchSuffix = $portVersion
-	lib:libnetpbm$secondaryArchSuffix = 11.91 compat >= 11
+	lib:libnetpbm$secondaryArchSuffix = 11.86 compat >= 11
```
[the remark this answers is not in the document]
Comment (Begasus) [reply media-libs/netpbm/netpbm-10.86.42.recipe:243]: Had been thinking on it, easy fix (will take care of it).

[the remark this answers is not in the document]
Comment (Begasus) [reply media-libs/netpbm/netpbm-10.86.42.recipe]: Done, thanks for catching!

File: media-libs/netpbm/netpbm-10.86.42.recipe [hunk 2]
```diff
@@ -231,16 +231,19 @@ COPYRIGHT="2001 411toppm: Steve Allen
...
-SOURCE_URI="https://github.com/t6/netpbm/releases/download/v$portVersion/netpbm-$portVersion.tar.xz"
-CHECKSUM_SHA256="dba419ffa3ac6af7d88c866abdeca64985c498a696906906e2dd719e10c7e12d"
+SOURCE_URI="https://sourceforge.net/projects/netpbm/files/super_stable/$portVersion/netpbm-$portVersion.tgz"
+CHECKSUM_SHA256="d4d2ca69d82b8d8518aee1d0a2f400664324afa8237859386764a41784d497f6"
 PATCHES="netpbm-$portVersion.patchset"
 
 ARCHITECTURES="all"
 SECONDARY_ARCHITECTURES="x86_gcc2 x86"
 
+libVersion="11.86"
```
Comment (pulkomandy) [inline media-libs/netpbm/netpbm-10.86.42.recipe:241]: Is it 11.86 or 10.86?

Comment (Begasus) [reply media-libs/netpbm/netpbm-10.86.42.recipe:241]: 11.86 :)
```
/Opslag/haikuports/media-libs/netpbm/work-10.86.42/sources/netpbm-10.86.42/lib> ls -la *.so*
lrwxrwxrwx 1 user root     15 jul. 22 09:43 libnetpbm.so -> libnetpbm.so.11
lrwxrwxrwx 1 user root     18 jul. 22 09:43 libnetpbm.so.11 -> libnetpbm.so.11.86
-rwxr-xr-x 1 user root 370088 jul. 22 09:43 libnetpbm.so.11.86
```

**Review by pulkomandy — APPROVED** [review]

**Comment by Begasus** [comment]
> GCC2 build failed with parse errors: https://build.haiku-os.org/buildmaster/master/x86_gcc2/logviewer.html?buildruns/6559/builds/83816.log, breaking also `latex2html` on 32 bits (from [report.txt](https://eu.hpkg.haiku-os.org/haikuports/master/x86_gcc2/current/report.txt)):
> 
> > requires "cmd:pnmcrop" of package "latex2html-2024-1" could not be resolved
> 
> Given that nothing seems to depend on `libnetpbm` on current HaikuPorts' tree... might be time to switch it to x86 only on 32 bits?

Fixed in https://github.com/haikuports/haikuports/pull/10738

**Pull request merged**